src/app/package.json

```diff
@@ -25,6 +25,8 @@
         "react-copy-to-clipboard": "5.0.1",
         "react-dom": "16.8.6",
         "react-hot-loader": "4.9.0",
+        "react-infinite": "0.13.0",
+        "react-infinite-any-height": "^2.3.0",
```

Comment:
Consider removing `^` here just to stay in tune with the other imports.

Reply:
Good idea. Will do.

Reply:
Pinned the `react-infinite-any-height` version in fixup a9c5d89